includes/constants.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
Could you store these constants outside of an includes files? (interface, business object..)